manim/scene/js_scene.py

```diff
@@ -109,7 +109,6 @@ def progress_through_animations(self):
                     logger.error(e)
             self.animation_finished.wait()
 
-    @scene.handle_play_like_call
```

Comment:
Question (non-blocking: what happened here?

Reply:
The JS renderer doesn't actually use a SceneFileWriter and now that it can handle its own rendering logic it doesn't need to call it.